src/components/avatar-button/avatar-button.tsx

```diff
@@ -0,0 +1,17 @@
+import type { FC, HTMLAttributes } from 'react'
+import { Avatar } from '../avatar/avatar'
+import { ElAvatarButton } from './styles'
+
+export interface AvatarButtonProps extends HTMLAttributes<HTMLButtonElement> {
+  label: string
+}
+
+export const AvatarButton: FC<AvatarButtonProps> = ({ label, ...props }) => {
+  return (
+    <ElAvatarButton aria-label="User Avatar" {...props}>
```

Comment:
> Also, this aria label is currently acting as a default value, because if an aria-label is provided in props, it'll override the value you're supply here. We should avoid doing this for defaults and instead leverage defaults (i.e. ({ 'aria-label: ariaLabel = 'blah', label, ...rest }))

ok, sure